subsys/bluetooth/host/Kconfig

```diff
@@ -702,6 +702,26 @@ config BT_DF_CTE_TX_AOD
 	  Enable support for antenna switching during CTE transmission.
 	  Also known as Angle of Departure mode.
 
+config BT_DF_VS_CL_IQ_REPORT_16_BITS_IQ_SAMPLES
```

Comment:
that's a long name, could `BT_DF_VS_CL_IQ_REPORT_16_BITS` be enough? given we have a help text and all.

Reply:
Actually I prefer long name instead of forcing every one to look into the definition to be sure what means this `16_BITS` in the option name.